pkgs/servers/matrix-conduit/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchFromGitLab, rustPlatform, }:
+rustPlatform.buildRustPackage rec {
+  pname = "matrix-conduit";
+  version = "unstable-2021-05-21";
+
+  src = fetchFromGitLab {
+    owner = "famedly";
+    repo = pname;
+    rev = "1aec80c57c02ef17475436c530b2ac2bc947b923";
+    sha256 = "sha256-bz+afiRaGQ5wHRVzEXNoaFuS36JU9sOGjQ/7xQtl458=";
```

Comment:
Please use the old base32 hashes for now.

Reply:
Why? Both hashes are actively in use in nixpkgs right now.

Reply:
Yeah and it is confusing and the notation isn't consistent because people refuse to stick to *stable* nix as the primary target. `sha256 = "sha256-..."` is odd enough. I am all for the new SRI hashes once we start migrating to `hash = "sha256-..."` everywhere.